ruma-events/tests/room_message.rs

```diff
@@ -26,12 +26,12 @@ use serde_json::{from_value as from_json_value, json, to_value as to_json_value}
 #[test]
 fn serialization() {
     let ev = MessageEvent {
-        content: MessageEventContent::Audio(AudioMessageEventContent {
+        content: MessageEventContent::new(MessageType::Audio(AudioMessageEventContent {
```

Comment:
Could use `.into()` here after 798cd49e. But I guess I should do it after this PR so it doesn't require yet another rebase 🙂